### ISSUE TITLE
Tag-110-ReportWriter-Term Type Shows Term_Group

### ISSUE THREAD

**Issue by rmehri** [NCIEVS/nci-protege5#125]
Application has PT for Term Type but in report shows as Term Group.  Please see attached snapshots and Report Writer.

Thanks,
Reza 

[ReportWriter .txt](https://github.com/NCIEVS/nci-protege5/files/646611/ReportWriter.txt)
![app_property](https://cloud.githubusercontent.com/assets/8367102/21108191/52f4e446-c062-11e6-9706-74f1cc0776f9.png)
![reportwriter](https://cloud.githubusercontent.com/assets/8367102/21108203/602652f8-c062-11e6-8f9e-78c6670dde14.png)

**Comment by bdionne** [comment]
Not sure this is a bug, the report writer is using the IRI fragments for the annotations, and the complex property table columns use the rdfs:label.

We can easily change it if the label is preferred

**Comment by rmehri** [comment]
I extended my testing and please attach zip file to play the video for more details.

**Comment by NCIEVS** [comment]
Yes, we should display the labels when rendering the concepts.  This is a new feature though as our production report writer doesn't display labels.  Will mark as enhancement for future release.  

The priority might fall even further to wontfix depending on the future need of this report type.

**Comment by bdionne** [comment]
feedback form editors needed

**Comment by NCIEVS** [comment]
Feedback from [EHD] and [NT]

"This is a low priority enhancement as it was submitted by Reza and not our editors. This particular issue may have been impacted by the change to ByCode for this qualifier"

**Comment by bdionne** [comment]
so for this one, we just need a decision on whether to continue using the `shortForm()` or switching to the label. It's a simple code change

cc: @NCIEVS

**Comment by hahndane** [comment]
I guess it should display the label for the qualifiers because the report displays the label for the properties (FULL_SYN rather than P90 etc.).

**Comment by singhik** [comment]
@bdionne : As per Liz, the report should display Labels f.e. FULL_SYN but the export file displays codes (P90) etc ? ?

![image](https://user-images.githubusercontent.com/36314371/129390457-a2979b8b-abf9-4826-9585-352854eb0883.png)

![image](https://user-images.githubusercontent.com/36314371/129390478-8698f75a-57c7-4569-ab04-72471f7ab842.png)

**Comment by bdionne** [comment]
@singhik ok, I'll look at this and discuss with @NCIEVS when we meet on Monday. 

For sure I didn't change anything wrt how complex properties are rendered, so if you see `P90` instead go `FULL_SYN` it may be due to how you have the rendered configured in the preferences.

What I did change is how the qualifiers are displayed, changing them from using what's called the `shortForm()` to using `rtfs:label` if one exists. This is what I thought this ticket is about, as in the title and Reza's original screenshots above. We'll review and if needed I'll change it to use `rtfs:label` everywhere.

Thanks.

**Comment by NCIEVS** [comment]
Hi @bdionne , it looks from [[EHD](https://github.com/NCIEVS/nci-protege5/issues/125#issuecomment-861666993)] that the qualifiers should use the labels as the properties were being displayed with "labels".  (Seems to me that the IRIs were being used for both, but this test was done when the NCIt was still [byName](https://github.com/NCIEVS/nci-protege5/issues/125#issuecomment-826752739), so those property "labels" were likely short forms of the IRI.)  We can go with labels for both qualifiers and properties as this is the expectation.

**Comment by bdionne** [comment]
@NCIEVS @singhik 

Ok, easy enough to change. Looking at the code it uses `shortForm()` everywhere, except for certain props like `code` and `preferred_name`. I thought they want `shortForm()` still as that gives you the code when it's part of the `IRI`.

I'll change it to use `rtfs:label` everywhere, which also means to test if in fact there is one. Should I default to the `IRI` fragment if the `rtfs:label` is not there?

We'll push another plugin version, after we see what else we uncover.

**Comment by NCIEVS** [comment]
@bdionne Yes, agreed, we should default to the short form if rdfs:label doesn't exist.

**Comment by singhik** [comment]
@bdionne  : 1.3.1 Edit Tab also does not display labels in Exports but codes , also 1.3.1 Edit Tab 'configure' export gets disabled after one export ,unless restarted 

![image](https://user-images.githubusercontent.com/36314371/130124541-85042cb0-6cb7-4bb6-99b5-04fc679d0e22.png)

![image](https://user-images.githubusercontent.com/36314371/130124586-8bf0672c-a827-4855-b525-9802bce9fd5d.png)

**Comment by bdionne** [comment]
@singhik I just pushed a [1.3.2](https://github.com/NCIEVS/nci-edit-tab/releases/tag/nci-edit-tab-1.3.2) that should work better. I'll test it with the full thesaurus tomorrow
